src/partitioner/local/MetisPartitioner.cpp

```diff
@@ -1009,3 +1009,125 @@ void MetisPartitioner::writeSerializedCompositeMasterFiles(std::string part) {
     masterFileMutex.unlock();
     partitioner_logger.log("Serializing done for central part " + part, "info");
 }
+
+void MetisPartitioner::setPathAndGraphID(int graphID) {
+    this->outputFilePath = Utils::getHomeDir() + "/.jasminegraph/tmp/" + std::to_string(graphID);
+    this->graphID = graphID;
+}
+
+void MetisPartitioner::loadFileChunks(string inputFilePath) {
```

Comment:
This function name is misleading. It seems it does not do any meaningful processing.

Reply:
This method is no longer there in the updated hdfs implementation.